src/status_im/ui/screens/wallet/send/redesigned_views.cljs

```diff
@@ -0,0 +1,98 @@
+            [taoensso.timbre :as log]
+            [reagent.core :as reagent]))
+
+(defview choose-amount-token []
+  (letsubs [{:keys [transaction modal? contact native-currency]} [:get-screen-params :wallet-choose-amount]
+            balance       [:balance]
+            prices        [:prices]
+            network       [:account/network]
+            all-tokens    [:wallet/all-tokens]
+            fiat-currency [:wallet/currency]
+            web3          [:get :web3]]
+    (let [chain (ethereum/network->chain-keyword network)]
+      [amount/render-choose-amount {:web3            web3
+                                    :balance         balance
+                                    :network         network
+                                    :chain           chain
+                                    :all-tokens      all-tokens
+                                    :modal?          modal?
+                                    :prices          prices
+                                    :native-currency native-currency
+                                    :fiat-currency   fiat-currency
+                                    :contact         contact
+                                    :transaction     transaction}])))
+
+(defview transaction-overview []
+  (letsubs [{:keys [transaction flow contact]} [:get-screen-params :wallet-txn-overview]
+            prices        [:prices]
+            network       [:account/network]
+            all-tokens    [:wallet/all-tokens]
+            fiat-currency [:wallet/currency]
+            web3          [:get :web3]]
+    (let [chain (ethereum/network->chain-keyword network)
+          native-currency (tokens/native-currency chain)
+          token (tokens/asset-for all-tokens
+                                  (ethereum/network->chain-keyword network) (:symbol transaction))]
+      [overview/render-transaction-overview {:transaction     transaction
+                                             :flow            flow
+                                             :contact         contact
+                                             :prices          prices
+                                             :network         network
+                                             :token           token
+                                             :native-currency native-currency
+                                             :fiat-currency   fiat-currency
+                                             :all-tokens      all-tokens
+                                             :chain           chain
+                                             :web3            web3}])))
+
+;; MAIN SEND TRANSACTION VIEW
+(defn- send-transaction-view [_opts]
+  (reagent/create-class
+   {:reagent-render (fn [opts] [recipient/render-choose-recipient opts])}))
+
+;; SEND TRANSACTION FROM WALLET (CHAT)
+(defview send-transaction []
+  (letsubs [transaction    [:wallet.send/transaction]
```

Comment:
It seems that `:wallet.send/transaction` subscription causes reloading of the screen right after rendering. In result text-input is focused twice and keyboard/view twitches.

Reply:
Good catch, I'll remove the subscription then. It's not even being used in the wallet flow

Reply:
> and keyboard/view twitches

at least on android real device